### ISSUE TITLE
Not the same error report in the web vs the terminal

### ISSUE THREAD

**Issue by flamendless** [koalaman/shellcheck#849]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2140
- My shellcheck version (`shellcheck --version` or "online"): 0.3.7 and online
- [X] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/your/interpreter
your script here
`git_push() {
  if [ $# -eq 0 ]; then
     echo USAGE:
     echo arg1 is the directory
     echo arg2 is the message
  elif [ $# -eq 1 ]; then
     correct=false
     case $1 in
	nightmare) cd ~/Projects/Nightmare || return; correct=true;;
        codes) cd ~/Documents/codes || return; correct=true;;
	brbl) cd ~/Projects/"GameJolt Sites"/brbl-bootstrap || return; correct=true;;
	sublime)  cd ~/.config/sublime-text-3/ || return; correct=true;;
	orbit) cd ~/Projects/orbit || return; correct=true;;
     esac
	if [ $correct == true ]; then
		git push origin master
		echo pushed
	else
		echo provide valid path
	fi
  else
	correct=false
	case $1 in
	  nightmare) cd ~/Projects/Nightmare || return; correct=true;;
	  codes) cd ~/Documents/codes || return; correct=true;;
	  brbl) cd ~/Projects/"GameJolt Sites"/brbl-bootstrap || return; correct=true;;
	  sublime) cd ~/.config/sublime-text-3/ || return; correct=true;;
	  orbit) cd ~/Projects/orbit || return; correct=true;;
	esac
	  if [ $correct == true ]; then
	  	git add -A
	  	git commit -m "$2"
	  	git push origin master
		echo pushed
	  else
		echo directory is wrong
	  fi
  fi
}`

```

#### Here's what shellcheck currently says:
In the web-based: nothing

in the terminal:  SC2140: The double quotes around this do nothing. Remove or escape them.

**Comment by nihilus** [comment]
Which interpreter are you using?

**Comment by flamendless** [comment]
bash - #!/bin/sh

**Comment by nihilus** [comment]
You should add a shebang that is '#!/bin/bash' then to your tests.

**Comment by flamendless** [comment]
Okay, that solves it
